### DISCUSSION TITLE
Documentación desactualizada y herrada en la página https://allurereport.org/docs/cypress/

### DISCUSSION THREAD

**Discussion by luijo-dev** [allure-framework/allure2#2743] [Documentation] [answered]
Se debe actualizara la configuración que se muestra en la página principal del framework porque viene con varios errores incluso desde los primeros pasos en la integración con cypress y la instalación con node.js donde se pide ejecuta como yarn dlx allure ... pero la consola nos dirá que dlx no existe, en cambio, la información que nos brinda el comando npx allure --help si que está al día. 
¿Les ha pasado?

**Comment by baev** [comment, marked as answer]
Thanks, we will do that. 

As for now, you should either:

1. Install Allure globally, e.g. by using instructions provided in https://allurereport.org/docs/install-for-nodejs/
2. Use `npx allure-commandline serve <path/to/allure/results>`
3. Install `allure-commandline` as a dev dependency and add a script to the package.json, e.g. `allure generate <path/to/results>` or `allure serve <path/to/results>`
